index.js

```diff
@@ -23,68 +26,72 @@ function getRandomNumber(min, max) {
 }
 
 function checkGuess() {
-  // Get value from guess input element
   const guess = parseInt(guessInput.value, 10);
   attempts = attempts + 1;
 
   hideAllMessages();
 
-  if (guess === targetNumber) {
-    numberOfGuessesMessage.style.display = '';
+  if (guess < guessInput.min) {
+    wayTooLow.style.display = "block";
+  } else if (guess > guessInput.max) {
+    wayTooHigh.style.display = "block";
+  } else if (guess === targetNumber) {
+    numberOfGuessesMessage.style.display = "";
     numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
-
-    correctMessage.style.display = '';
-
+    correctMessage.style.display = "";
     submitButton.disabled = true;
     guessInput.disabled = true;
-  }
 
-  if (guess !== targetNumber) {
+    if (attempts === maxNumberOfAttempts) {
+      submitButton.disabled = true;
+      guessInput.disabled = true;
+    }
+  } else {
     if (guess < targetNumber) {
-      tooLowMessage.style.display = '';
+      tooLowMessage.style.display = "block";
```

Comment:
I like that you're setting these back to `display: block` when you want to render them again, but I noticed you only made that change in some places and some were left being set back to an empty string. Could be worth keeping these consistent, but in the a app I think we might want to add and remove a `.hidden` class instead.